ocaml/xapi/repository_helpers.ml

```diff
@@ -717,6 +717,146 @@ let get_updates_from_repoquery repositories =
   in
   new_updates @ updates
 
+(* Parser module for the output of command yum upgrade *)
+module YumUpgradeOutput = struct
+  open Angstrom
+
+  let consume = Consume.Prefix
+
+  let is_eol = function '\n' -> true | _ -> false
+
+  let is_eol_tab_or_space = function '\n' | '\t' | ' ' -> true | _ -> false
```

Comment:
This is traditionally called white space; in programming languages tokens are separated by white space and it is often not significant.

Reply:
This is (Astring.Char.Ascii.is_white](https://ocaml.org/p/astring/latest/doc/Astring/Char/Ascii/index.html#val-is_white)